hyf-core-values.md

```diff
@@ -2,22 +2,23 @@
 
 ## Learning
 * Our program is about problem solving and learning how to learn.
-* Take responsibility for your own learning process.
-* Tell us and your fellow students what works for you and what doesn't when it comes to learning.
-* Be active and communicate about things that are not clear.
-* Help us and our teachers improve by giving regular feedback.
+* Take responsibility for your own learning process, what you put into this program is what you get out.
+* Learning how to code is hard. Don't pretend to understand things if you don't, just ask questions. We love questions. 
```

Comment:
Maybe instead: If you are having trouble understanding certain concepts, ask questions. We love questions.